gridstatus/tests/source_specific/test_miso_api.py

```diff
@@ -135,3 +135,73 @@ def test_get_lmp_real_time_5_min_ex_post_final_date_range(self):
 
         assert df["Interval Start"].min() == start
         assert df["Interval End"].max() == end
+
+    """get_interchange_hourly"""
+
+    def _check_interchange_hourly(self, data: pd.DataFrame):
+        assert data.columns.tolist() == [
+            "Interval Start",
+            "Interval End",
+            "Net Scheduled Interchange Forward",
+            "Net Scheduled Interchange Real Time",
+            "Net Scheduled Interchange Delta",
+            "MHEB Scheduled",
+            "MHEB Actual",
+            "ONT Scheduled",
+            "ONT Actual",
+            "SWPP Scheduled",
+            "SWPP Actual",
+            "TVA Scheduled",
+            "TVA Actual",
+            "AECI Scheduled",
+            "AECI Actual",
+            "SOCO Scheduled",
+            "SOCO Actual",
+            "LGEE Scheduled",
+            "LGEE Actual",
+            "PJM Scheduled",
+            "PJM Actual",
+            "OTHER Scheduled",
+            "SPA Actual",
+        ]
+
+        assert data["Interval Start"].dtype == "datetime64[ns, EST]"
+        assert data["Interval End"].dtype == "datetime64[ns, EST]"
+
+        for col in data.columns:
+            if col not in ["Interval Start", "Interval End"]:
+                assert data[col].dtype == "float64"
+
+    def test_get_interchange_hourly_today(self):
+        with api_vcr.use_cassette("test_get_interchange_hourly_today"):
+            df = self.iso.get_interchange_hourly("today")
+
+        self._check_interchange_hourly(df)
+
+        assert df["Interval Start"].min() == self.local_start_of_today()
+        # Scheduled data extends to the end of the day
+        assert df["Interval Start"].max() == self.local_start_of_today() + pd.Timedelta(
+            hours=23,
+        )
+
+    def test_get_interchange_hourly_date_range(self):
+        start = (
+            self.local_start_of_today()
+            - pd.DateOffset(days=20)
+            + pd.DateOffset(hours=10)
+        )
+        end = start + pd.DateOffset(days=2, hours=4)
+
+        with api_vcr.use_cassette(
+            "test_get_interchange_hourly_{start:%Y%m%d}_{end:%Y%m%d}",
```

Comment:
```suggestion
            f"test_get_interchange_hourly_{start:%Y%m%d}_{end:%Y%m%d}",
```
Need a f-string here, and a `.yaml` filetype on the fixture cassettes generally in this file. I think it technically works without it but shows up weird in `fixtures/`

![CleanShot 2025-06-14 at 20 10 44@2x](https://github.com/user-attachments/assets/63da8f11-9ccf-49b5-ae38-b6c025bbaf56)